contracts/contracts/interfaces/l1/ILineaRollup.sol

```diff
@@ -113,12 +96,12 @@ interface ILineaRollup {
    * @dev l1RollingHashMessageNumber is the calculated message number on L2 that is expected to match the existing L1 rolling hash.
    * This value will be used along with the stored last finalized L2 calculated message number in the public input.
    * @dev l2MerkleTreesDepth is the depth of all l2MerkleRoots.
-   * @dev l2MerkleRoots is an array of L2 message Merkle roots of depth l2MerkleTreesDepth between last finalized block and finalSubmissionData.finalBlockInData.
+   * @dev l2MerkleRoots is an array of L2 message Merkle roots of depth l2MerkleTreesDepth between last finalized block and finalSubmissionData.finalBlockNumber.
    * @dev l2MessagingBlocksOffsets indicates by offset from currentL2BlockNumber which L2 blocks contain MessageSent events.
    */
   struct FinalizationDataV3 {
     bytes32 parentStateRootHash;
-    uint256 finalBlockInData;
+    uint256 finalBlockNumber;
```

Comment:
```suggestion
    uint256 endBlockNumber;
```

`final` leads me to the finality concept, so I would favor interval terminology, start/end

Reply:
addressed in new commit